packages/component/src/SendBox/AttachmentBar/Preview/FileIcon.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { memo } from 'react';
+
+type FileIconProps = Readonly<{
+  size?: 'large' | 'small' | undefined;
+}>;
+
+const FileIcon = ({ size }: FileIconProps) =>
```

Comment:
Why we don't use the image masker approach here?

Reply:
I forgot we have that component and didn't test dark mode. 🫢

Reply:
Oh, another thing, the icon is a brand, think about it is a Word document icon

So, there is nothing like dark mode and masker won't work because: 1. it should not be colored, 2. it should always white in color.

Reply:
I think we should adjust the component with the support of multicolor icons, so we're able to swap them which CSS properties. 

It'd be even better if we could decide inside CSS which variant to render e.g. if full color icon is specified, should we render it instead replace f masked?

Reply:
Some idea I have in mind:

```css

.webchat [data-glyph=send] {
  --glyph-image: url(...); /* full color image to be used across Web Chat*/
}

.webchat .send-box [data-glyph=send] {
  --glyph-mask: url(...); /* used mask with plain color only for send box area send glyph */
}

/* Some EEE neats */

.webchat .variant-dark [data-glyph=send] {
  --glyph-mask: url(...); /* use mask with custom plain color only for a custom dark variant (not supported yet */
  --glyph-color: purple;
}

.webchat .send-box [data-glyph=send] {
  --glyph-size: 14px; /* This says: I want smaller glyph for send box area */
}

.webchat [data-glyph] {
  --glyph-image-mask: url("data...<circle ... />"); /* This says: I want any glyph image to be masked with circle */
}


```

Reply:
Here https://bolt.new/~/sb1-z823d3hr

Reply:
Done.